### DISCUSSION TITLE
How to resume a Lora training?

### DISCUSSION THREAD

**Discussion by ra-ra-ra-ra** [bmaltais/kohya_ss#1781] [Q&A]
Hello, for my first ever attempt to do a Lora, I started a training that would last 90 hours, but after 15/20 hrs my electricity went down for some maintenance work which I was unaware, and now I have only 2 Loras inside my folders (because I saved every 1 epoch).. Is there a way to resume the training from that second lora (they needed to be around 10 in total) or do I have to start from scratch?
I saw that voice:
LoRA network weights
Path to an existing LoRA network weights to resume training from 
but choosing the second Lora created doesn't let me resume it, I'm having an issue about permission denied on the terminal

`PermissionError: [Errno 13] Permission denied: 
'C:\\Users\\{ USER_NAME }\\Downloads\\TRAIN LORA\\TRAIN FIELD\\{ MODEL_NAME}\\model'`

**Comment by 5KilosOfCheese** [comment]
To resume training you must first enable the `save training state`. 
![kuva](https://github.com/bmaltais/kohya_ss/assets/117771081/c0f6e041-715e-4b22-beb8-8cc96ce0bf6d)
This saves the training state at the save interval set in `save every N epoch`.
![kuva](https://github.com/bmaltais/kohya_ss/assets/117771081/981c327a-3bff-4347-bba8-45da06352998)

To resume training, you must select the folder in your model folder which has "-state" in it's name. This is done by selecting that folder within the GUI or pasting it's path. This folder contains the actual training. The output file ckpt/safetensor is just the end product derived from this.
